### PR TITLE
docs: remove Atom as a recommended code editor

### PR DESCRIPTION
**Pull request by RafaelDavisH**
Checklist:

- [X] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [X] I have read and followed the [how to open a pull request 
guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [ ] I have tested these changes either locally on my machine, or GitPod.

### CODE REVIEW

File: docs/how-to-setup-freecodecamp-locally.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ Once you have the prerequisites installed, you need to prepare your development
 
 3. Install a code editor of your choice.
```
Comment (raisedadead) [inline docs/how-to-setup-freecodecamp-locally.md:84]:
```suggestion
```

File: docs/how-to-setup-freecodecamp-locally.md [hunk 2]
```diff
@@ -83,7 +83,7 @@ Once you have the prerequisites installed, you need to prepare your development
...
 
-   We highly recommend using [Visual Studio Code](https://code.visualstudio.com/) or [Atom](https://atom.io/). These are great, free and open source code editors.
+   Use your favorite text editor, we recommend [Visual Studio Code](https://code.visualstudio.com/), it's free and open source.
```
Comment (raisedadead) [inline docs/how-to-setup-freecodecamp-locally.md:86]:
```suggestion
3. Install a code editor of your choice, we recommend [Visual Studio Code](https://code.visualstudio.com/), it's free and open source.
```

**Review by raisedadead — DISMISSED** [review]
Sorry this is nitpick but the reading it as a whole sounds repetitive, we can make it slightly more succint.

File: docs/how-to-setup-freecodecamp-locally.md [hunk 3]
```diff
@@ -81,9 +81,8 @@ Once you have the prerequisites installed, you need to prepare your development
 
 2. (Optional but recommended) [Set up an SSH Key](https://help.github.com/articles/generating-an-ssh-key/) for GitHub.
 
-3. Install a code editor of your choice.
 
-   We highly recommend using [Visual Studio Code](https://code.visualstudio.com/) or [Atom](https://atom.io/). These are great, free and open source code editors.
+3. Install a code editor of your choice, we recommend [Visual Studio Code](https://code.visualstudio.com/), it's free and open source.
```
Comment (naomi-lgbt) [inline docs/how-to-setup-freecodecamp-locally.md:85]:
```suggestion
3. Install a code editor of your choice. We recommend [Visual Studio Code](https://code.visualstudio.com/), it's free and open source.
```

My turn to nitpick - this is a comma splice (:

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

File: docs/how-to-setup-freecodecamp-locally.md (hunk 3 above)
Comment (RafaelDavisH) [inline docs/how-to-setup-freecodecamp-locally.md:85]:
```suggestion
3. Install a code editor of your choice. Nevertheless, we recommend [Visual Studio Code](https://code.visualstudio.com/), it's free and open source.
```
How about this?

Comment (naomi-lgbt) [reply docs/how-to-setup-freecodecamp-locally.md:85]: `Nevertheless` is just added fluff here, IMHO.

Comment (RafaelDavisH) [reply docs/how-to-setup-freecodecamp-locally.md:85]: We initially say 'You can install the code editor of your choice', but then we go very (IMO) cut and dry with our suggestion. Not a big deal of course - we can leave it as how Noami suggested. 

Maybe there is a less 'fluffy' word we can use to transition to our recommendation?

Comment (naomi-lgbt) [reply docs/how-to-setup-freecodecamp-locally.md:85]:
```suggestion
3. Install a code editor of your choice. If you aren't sure which one to use, we recommend [Visual Studio Code](https://code.visualstudio.com/) — it's free and open source.
```

If we want a transition...

**Review by naomi-lgbt — APPROVED** [review]

**Review by scissorsneedfoodtoo — APPROVED** [review]

**Pull request merged**